### PR TITLE
Remove private key login feature

### PR DESCRIPTION
**Pull request by kjhman21**

### CODE REVIEW

File: src/components/KlayFaucet.js [hunk 1]
```diff
@@ -83,6 +77,17 @@ class KlayFaucet extends Component<Props> {
     });
   }
 
+  changeWallet = (e) => {
+    if(e.target.value) {
+      this.wallet.address = e.target.value
```
Comment (jeonghonah) [inline src/components/KlayFaucet.js:82]: minor comment.
It would be nice if wallet address verification is added.

**Review by jeonghonah — APPROVED** [review]
LGTM

File: src/components/KlayFaucet.js (hunk 1 above)
Comment (kjhman21) [reply src/components/KlayFaucet.js:84]: good suggestion. @jeonghonah Could you please take another look?

**Review by jeonghonah — APPROVED** [review]
LGTM

**Pull request merged**